labels.json

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
It looks like labels can now have descriptions. Seems worth adding those for the `Status:`, `Priority:`, and `Type:` labels.